test/models/concerns/environment_extensions_test.rb

```diff
@@ -13,7 +13,14 @@ def setup
       @env = katello_environments(:dev)
       @content_view = katello_content_views(:library_dev_view)
       @content_view_puppet_env = katello_content_view_puppet_environments(:library_view_puppet_environment)
-      Environment.create!(:name => "env_for_test", :content_view_puppet_environment => @content_view_puppet_env)
+      @environment = Environment.new
```

Comment:
is this because name and content_view_puppet_environment aren't in attr_accessible? if so, why not just add them?